### PR TITLE
DMG-1016: Update Census Docs on Sync Triggers

### PR DESCRIPTION
**Pull request by padmaashini**
Following https://github.com/sutrolabs/giza/pull/9574, updating docs to reflect display of sync triggers through Management API.

### CODE REVIEW

**Comment by utD9FeifNY6fLo35PfwlcznLtzQdMpBpXqBFeZ2** [comment]
<p><a href="https://linear.app/getcensus/issue/DMG-1016/update-census-external-docs-on-management-api-changes-for-sync">DMG-1016 Update Census External Docs on Management API Changes for Sync Triggers</a></p>

**Review by sean-lynch — APPROVED** [review]
Looks great!

File: basics/developers/api/syncs.md [hunk 1]
```diff
@@ -337,7 +354,7 @@ curl --location --request POST 'https://app.getcensus.com/api/v1/syncs' \
...
-| trigger\_on\_dbt\_cloud\_rebuild                  | Whether or not this sync should trigger on a DBT cloud rebuild.                                                                                                                                                                                                                                                                                         |
+| triggers                                          |  Specify dbt Cloud job id and project id and/or Fivetran job id and job name to trigger Census syncs after dbt Cloud jobs and/or Fivetran syncs/transforms successfully complete.                                                                                                                                                                                                                                                                                          |
```
Comment (sww7tKLNrYaeUVT9kydWxjwaCX51qprgRz1Ldsv) [inline basics/developers/api/syncs.md]: Is there any more detail you can add about where the ids/names can be pulled from in Fivetran/dbt?

Comment (CDqxQ9ZYks1B4nszBztVqgfUOg0h7T2r3Ma9urG) [reply basics/developers/api/syncs.md]: for sure! added some more info :)

**Pull request merged**